reference/formats/message_format.rst

```diff
@@ -223,7 +223,7 @@ handle pluralization in your messages (e.g. ``There are 3 apples`` vs
         num_of_apples: >-
             {apples, plural,
                 =0    {There are no apples}
```

Comment:
No directly related to your contribution ... but in my apps I just use `0`, `1`, `other`, etc. instead of `=0`, `=1`, `other` and it works as intended. So, is the `=` required? If it's optional, do we want to keep it? Thanks.

Reply:
The doc said:
```
    // prints "There is one apple..."
    echo $translator->trans('num_of_apples', ['apples' => 1]);
```

But with rule:
```
{apples, plural, =0 {There are no apples} one {There is one apple...} other {There are # apples!}}
```

it prints `There are 1 apples`.

It was a mistake, or I just did wrong with the usage?